packages/frontend/src/pages/Signup.tsx

```diff
@@ -77,14 +138,17 @@ const Signup: FC = () => {
                 <LogoWrapper>
                     <Logo src={LightdashLogo} alt="lightdash logo" />
                 </LogoWrapper>
-                <CardWrapper elevation={2}>
-                    {inviteLinkQuery.error ? (
-                        <NonIdealState
-                            title={inviteLinkQuery.error.error.message}
-                            icon="error"
-                        />
-                    ) : (
-                        <>
+                {inviteLinkQuery.error ? (
+                    <ErrorCard
+                        title={
+                            inviteLinkQuery.error.error.name === 'ExpiredError'
+                                ? 'This invite link has expired 🙈'
+                                : inviteLinkQuery.error.error.message
+                        }
+                    />
+                ) : isLinkFromEmail ? (
```

Comment:
should this be `!isLinkFromEmail` ? 
This screenshot is from the e2e test fail
![Settings - Invites -- Should invite user (failed)](https://user-images.githubusercontent.com/9117144/173877577-8bb782f5-88b3-40ab-bebb-76fb1bc7717e.png)
ing